app/api/transformers/MediaItemTransformer.php

```diff
@@ -20,15 +20,12 @@ public function transform($mediaItemAndPlaylist, array $options) {
 		
 		$scheduledPublishTime = $mediaItem->scheduled_publish_time->timestamp;
 		
-		$coverArtUris = null;
-		if (!is_null($playlist)) {
-			$coverArtResolutions = Config::get("imageResolutions.coverArt");
-			
-			$coverArtUris = [
-				"thumbnail"		=> $playlist->getMediaItemCoverArtUri($mediaItem, $coverArtResolutions['thumbnail']['w'], $coverArtResolutions['thumbnail']['h']),
-				"full"			=> $playlist->getMediaItemCoverArtUri($mediaItem, $coverArtResolutions['full']['w'], $coverArtResolutions['full']['h']),
-			];
-		}
+		$coverArtResolutions = Config::get("imageResolutions.coverArt");
+		
+		$coverArtUris = [
+			"thumbnail"		=> $playlist->getMediaItemCoverArtUri($mediaItem, $coverArtResolutions['thumbnail']['w'], $coverArtResolutions['thumbnail']['h']),
```

Comment:
Did you try this locally? I'm not sure how this will work if `$playlist` is `null`.

One option would be defaulting `$playlist` to one of the media items it is in using 

https://github.com/LA1TV/Website/blob/43c623454e8bd5bd169ea9c887418c1d92883072/app/models/MediaItem.php#L401-L420

if it is null

Reply:
i should of looked, nvm 😆